requirements-server.txt

```diff
@@ -0,0 +1,15 @@
+# Dependencies for notebook server and interaction with JupyterHub
+jupyterhub==1.4.0
+jupyterlab==3.0.14
+notebook==6.3.0
+# Dependencies for jupyter widgets
```

Comment:
do we need all of these?

Reply:
Unfortunately yes. They are here just for activation purposes.